### ISSUE TITLE
Angular App (Webform) Changes

### ISSUE THREAD

**Issue by m-prodan** [bcgov/foi-flow#3164]
Title of ticket: Angular App (Webform) Changes

#### Description
With the new Cabinet - there are some changes that need to be made to the angular webform, including conditional logic changes as some assumptions will have changed around how application fees are handled.

Name changes should apply to both the online form, the email submitted and the receipt generated.

Net new Ministries will follow the same logic for general request application fees - requests directed to them will incur a 10 dollar application fee.

#### Dependencies
N/A

#### DOD - Simple Name Changes:
- [x] Advanced Education and Skills training is updated to: Post-Secondary Education and Future Skills
- [x] Land, Water and Resource Stewardship is updated to: Water, Land and Resource Stewardship
- [x] Jobs, Economic Recovery and Innovation is updated to: Jobs, Economic Development and Innovation
- [x] Attorney General (and Minister Responsible for Housing) is updated to: Attorney General
- [x] Energy, Mines and Low Carbon Innovation (and Minister Responsible for the Consular Corps of British Columbia) is updated to: Energy, Mines and Low Carbon Innovation
- [x] Forests is updated to: Forests (and Minister Responsible for Consular Corps)
- [x] Environment and Climate Change Strategy (and Minister Responsible for Translink) is updated to: Environment and Climate Change Strategy

#### DOD - REMOVE:
- [x] REMOVE Conditional application fee logic and selection of Emergency Management BC - they are no longer a subsidiary of public safety and solicitor general and will become a standalone Ministry

#### DOD - NEW Public Bodies/Ministries:
- [x] ADD NEW Ministry: "Emergency Management and Climate Readiness"
- [x] ADD NEW Ministry: "Housing"

#### DOD - BC Services Card URL Updates:
On this page:
![image.png](https://images.zenhubusercontent.com/6081a4c82f0c363dddb9c1a0/f3d97c18-c57e-4444-bf18-65807ea2c75b)

- [x] Update language/link: FROM: "First time BC Services Card Users: You need to verify your identity. Please set up your email when prompted.

TO: First time BC Services Card users: you will [need to set up a BC Services Card account](https://www2.gov.bc.ca/gov/content?id=8BB78F5239484C7D93015173C3AD1BD6) and verify your identity. Please input your email address when prompted.

- [x] Update link for:
![image.png](https://images.zenhubusercontent.com/6081a4c82f0c363dddb9c1a0/56ea50b7-4e32-40c7-8eb6-c9e5cfc9024d)

TO: https://www2.gov.bc.ca/gov/content?id=B2B3A21E797A421A8FD39EEA86E245D6

**Comment by m-prodan** [comment]
Updates to online webform per new cabinet being sworn in today.

@arielleandrews - do you see any gaps? Anything not listed here is assumed to not be changing at all

cc: @lmullane @liseandtea @KyEggleston @abin-aot @JHarrietha-AOT

**Comment by lmullane** [comment]
@abin-aot, when you have time, could you add an estimate.

**Comment by abin-aot** [comment]
@lmullane done! - lets start with 8 . if required , I will inform and increase later

**Comment by arielleandrews** [comment]
Added one more name change. Looks good to go now! @m-prodan @lmullane @abin-aot

**Comment by m-prodan** [comment]
@arielleandrews Thank you!

@abin-aot - I have now included the URL/Text changes requested by the BCSC team.

**Comment by lmullane** [comment]
@abin-aot, these are the new org codes for the 2 new ministries:

HSG = Housing
EMC = Emergency Management and Climate Readiness

Note, that from a reporting perspective all closed and inflight requests will remain with the original ministry (e.g. EMBC requests from before Dec 7th will remain PSSG).

New requests received from December 7th onwards will be under the new ministry (e.g., EMC).

Let me know if you have any questions?

cc @m-prodan 

cc @m-

**Comment by lmullane** [comment]
For the future application, when an org change has occurred we need to keep sign off flexible. Sometimes the sign off will be done by the executive in the original ministry, sometimes by the executive in the new ministry.

 In other words, we should not limit who is assigned as a reviewer based on the org code for the request.

**Comment by lmullane** [comment]
Hi @abin-aot, I have confirmed everything is working as expected.

However as above, the ministries should be listed alphabetically. Could you make that change?

**Comment by abin-aot** [comment]
@lmullane Yes, made that change as discussed. But just to indicate , earlier sorting was based on code , certain ministries was added in the middle - for some reason  and for particularly for personal requests - Certain ministries like MCFD and Solicitor General will be come on top based on the previous selections on it.  Now, I made it sorted as per the "NAME" of the public body and for personals - indicative ministries will be on top , then follows the alphabetical order. Make sense?
